docs/reference/kubectl/overview.md

```diff
@@ -192,6 +192,29 @@ NAME           RSRC
 submit-queue   610995
 ```
 
+#### Server-side columns
+
+`kubectl` supports receiving specific column information from the server about objects.
+This means that for any given resource, the server will return columns and rows relevant to that resource, for the client to print.
+This allows for consistent human-readable output across clients used against the same cluster, by having the server encapsulate the details of printing.
+
+All functionality of the `get` command is unaffected by using this feature.
+Clients are still able to return sorted output, perform template printing, print custom-columns, etc.
```

Comment:
This is too bold of a statement, we're actually discussing changing this so I'd drop both of these sentences.